gem2rpm/scl.spec.erb

```diff
@@ -12,6 +12,9 @@ config.rules[:ignore] << '.document'
 config.rules[:ignore] << 'Appraisals'
 config.rules[:ignore] << 'gemfiles'
 config.rules[:ignore] << 'appveyor.yml'
+config.rules[:ignore] << 'TODO'
+config.rules[:ignore] << 'sample'
+config.rules[:ignore] << 'web'
```

Comment:
In https://src.fedoraproject.org/rpms/rubygem-racc/blob/rawhide/f/rubygem-racc.spec they're shipped in the doc pacakge. I'm not sure we should ignore them.